spec.html

```diff
@@ -40918,7 +40922,7 @@ <h1>Pattern Semantics</h1>
           1. Return _m_.
         </emu-alg>
 
-        <p>Assertion (<emu-xref href="#sec-assertion"></emu-xref>) evaluation rules for the <emu-grammar>Assertion :: `(` `?` `=` Disjunction `)`</emu-grammar> and <emu-grammar>Assertion :: `(` `?` `!` Disjunction `)`</emu-grammar> productions are also used for the |QuantifiableAssertion| productions, but with |QuantifiableAssertion| substituted for |Assertion|.</p>
+        <p>Assertion (<emu-xref href="#sec-assertion"></emu-xref>) evaluation rules for the <emu-grammar>Assertion :: `(` `?` `=` Disjunction `)`</emu-grammar>, <emu-grammar>Assertion :: `(` `?` `!` Disjunction `)`</emu-grammar>, <emu-grammar>Assertion :: `(` `?` `&lt;=` Disjunction `)`</emu-grammar>, and <emu-grammar>Assertion :: `(` `?` `&lt;!` Disjunction `)`</emu-grammar> productions are also used for the |QuantifiableAssertion| productions, but with |QuantifiableAssertion| substituted for |Assertion|.</p>
```

Comment:
Do we actually want to enable lookbehind in QuantifiableAssertion?

Reply:
We should make sure to have a test262 test one way or the other.

Reply:
Just a note: I confirmed it on Chrome 65 since Annex B is the section for browser compatibility. Lookbehind assertions are quantifiable on the Chrome.

Reply:
Good to know, though I think it's still within our ability to make this choice one way or another.

cc @hashseed @schuay